resim/msg/fuzz_helpers.hh

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 ReSim, Inc.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+#pragma once
+
+#include <cstdint>
+#include <random>
+
+#include "resim/msg/header.pb.h"
+#include "resim/msg/transform.pb.h"
+#include "resim/testing/fuzz_helpers.hh"
+#include "resim/testing/random_matrix.hh"
+#include "resim/transforms/proto/se3_to_proto.hh"
+#include "resim/utils/uuid.hh"
+
+namespace resim::msg {
+
+template <typename Rng>
```

Comment:
Do these have to be defined in header? No need to change either way, just interested.

Reply:
They do unless we want to specialize them only for a certain set of Rngs.